docs/setup/upgrade/logging-configuration-migration.asciidoc

```diff
@@ -1,13 +1,32 @@
 [[logging-configuration-migration]]
 == Logging configuration migration
 
+[float]
+=== Prepare for logging in 8.0
+
+The entire logging system changed to offer a more consistent logging behavior with {es}. The current system differs from the more simplistic, deprecated system and allows you the freedom to customize what, how and where {kib} outputs log messages. While for logging based use cases, we recommend the <<logging-service, logging service guide>>, there are some important considerations:
```

Comment:
nit: Maybe specifically call out `log4j 2` here?

> The entire logging system changed to offer a more consistent logging behavior with {es}, which uses the popular `log4j 2` for managing logs.

Reply:
nit: For whatever reason my brain isn't grokking this line: 

> While for logging based use cases, we recommend the <<logging-service, logging service guide>>, there are some important considerations

Not super sure what you intended here, but maybe something like this?

> The best place to learn about the new logging system is our logging service guide. However, there are some important considerations when you are working from migrating from the old to the new system:

Reply:
I was trying to say that we recommend going through the logging migration guide if you rely on Kibana logging. Your wording is clearer, thanks!